ext/web/timers.rs

```diff
@@ -79,13 +80,15 @@ pub fn op_timer_handle(state: &mut OpState) -> ResourceId {
 /// [`TimerHandle`] resource given by `rid` has been canceled.
 ///
 /// If the timer is canceled, this returns `false`. Otherwise, it returns `true`.
-#[op(deferred)]
+#[op2(async(lazy))]
 pub async fn op_sleep(
   state: Rc<RefCell<OpState>>,
-  millis: u64,
-  rid: ResourceId,
+  #[bigint] millis: u64,
```

Comment:
I need to implement `#[number]` for these so we can avoid a v8::BigInt. I don't think it should block anything and it's not a crazy-difficult patch.